Tests/LoggingTests/TestLogger.swift

```diff
@@ -252,11 +252,11 @@ public class MDC {
     }
 
     private var threadId: Int {
-      #if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
-          return Int(pthread_mach_thread_np(pthread_self()))
-      #else
-          return Int(pthread_self())
-      #endif
+        #if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
```

Comment:
shouldn't we match the formatting that Xcode does by default? For this one, Xcode does

```
    private var threadId: Int {
        #if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
        return Int(pthread_mach_thread_np(pthread_self()))
        #else
        return Int(pthread_self())
        #endif
    }
```

Reply:
i have no opinion on the what format we should use. just want to make sure its consistent. currently we mostly use `swiftformat --self insert --patternlet inline --stripunusedargs unnamed-only --comments ignore` but happy apply other settings if the team prefers something else. just let me know or put a follow up PR to "fix"